features/affiliate-login-temporarily-disable.feature

```diff
@@ -0,0 +1,18 @@
+@issue-875
+Feature: Go to login page and affiliate login options will not seen
+  As an author,
+  I do not want to see the affiliate login options in the login page
```

Comment:
Hi @kencho51 

Gherkin syntax is:
>As ...
>I want ...
>So that ...

so, try to reformulate the story, such that "I do not want ..." becomes "I want ..."